include/circt/Support/InstanceGraphInterface.td

```diff
@@ -49,6 +49,14 @@ def InstanceGraphInstanceOpInterface : OpInterface<"InstanceOpInterface"> {
       return ::mlir::ArrayAttr::get(attr.getContext(), {attr});
     }]>,
   ];
+
+  let verify = [{
+    if (mlir::cast<ConcreteOp>($_op)
+            .getReferencedModuleNamesAttr()
+            .size())
+      return success();
+    return $_op->emitOpError("has empty list of referenced modules");
```

Comment:
Mega-nit: error language that is more affirmatively states what the problem is or what the expectation is would be better than an observation (with the implication that this is the problem or the expectation was the inverse). Consider either:

- "expected a non-empty list of referenced modules"
- "must have a non-empty list of referenced modules"